sidebars.js

```diff
@@ -200,7 +201,11 @@ module.exports = {
       label: 'Admin Portal',
       collapsible: true,
       link: { type: 'doc', id: 'admin-portal/overview' },
-      items: ['admin-portal/enterprise-sso', 'admin-portal/directory-sync'],
+      items: [
+        'admin-portal/enterprise-sso',
+        'admin-portal/directory-sync',
+        'admin-portal/federated-saml',
```

Comment:
This should be a section under Enterprise SSO as opposed to separate (should reflect our Admin Portal).